### DISCUSSION TITLE
Is there the validation dataset?

### DISCUSSION THREAD

**Discussion by tphankr** [FETS-AI/Challenge#28] [General]
Please, help me have the validation dataset. Where link can I download it?
![image](https://user-images.githubusercontent.com/37217354/121932254-aa961500-cd12-11eb-8ab9-4a7c8bd88636.png)

Thank you,

**Comment by sarthakpati** [comment]
Hi @tphankr,

Thank you for your interest in the FeTS Challenge!

We are working to make the validation data available tonight by midnight (EST).

Cheers,
Sarthak on behalf of the FeTS Challenge Organizing Committee

**Comment by tphankr** [comment]
Thank you, the CBICA team.
